RampUpAuto/src/test/java/seleautomation/com/RampUpAuto/GitHubSighInTest.java

```diff
@@ -0,0 +1,145 @@
+package com.Github;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.testng.annotations.*;
+import org.testng.Assert;
+
+
+class GitHubSignInPage {
+
+    WebDriver driver;
+
+    GitHubSignInPage(WebDriver driver) {
+
+        this.driver = driver;
+
+    }
+
+    public void signIn(String username, String password) {
+
+        WebElement usernameInput = driver.findElement(By.id("login_field"));
+
+        WebElement passwordInput = driver.findElement(By.id("password"));
+
+        WebElement signInButton = driver.findElement(By.name("commit"));
+
+        usernameInput.sendKeys(username);
+
+        passwordInput.sendKeys(password);
+
+        signInButton.click();
+
+    }
+
+    public boolean isSignInSuccessful() {
+
+        WebElement userAvatar = driver.findElement(By.className("avatar"));
+
+        return userAvatar.isDisplayed();
+
+    }
+
+    public boolean isErrorMessageDisplayed() {
+
+        WebElement errorMessage = driver.findElement(By.cssSelector(".flash-error"));
+
+        return errorMessage.isDisplayed();
+
+    }
+
+
+}
+
+public class GitHubSignInTest {
+    WebDriver driver;
+    public GitHubSignInPage signInPage;
+
+
+    @BeforeTest
+    @Parameters("browser")
+    public void create(@Optional("chrome")String browser) throws Exception {
```

Comment:
Regarding the POM also you can create other class for testing like for example `signInTesting` and add these functions `create` , `DBgithubSignin` and `githubSignIn`